cmd/prometheus/main.go

```diff
@@ -498,6 +503,13 @@ func main() {
 		})
 	}
 
+	closer, err := initTracing(logger)
+	if err != nil {
+		level.Error(logger).Log("msg", "Unable to init tracing", "err", err)
+		os.Exit(1)
```

Comment:
exit 2 is what we do for such kind of errors

Reply:
Updated